<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArtikelErfassen</class>
 <widget class="QDialog" name="ArtikelErfassen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Artikel erfassen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="btSpeichern">
     <property name="text">
      <string>Speichern</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QCheckBox" name="chTglB">
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Täglicher Bedarf</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="3">
    <widget class="QCheckBox" name="chCodeCheck">
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Barcode Check</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Barcode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QLineEdit" name="tbBarcode">
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::StrongFocus</enum>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QLineEdit" name="tbName">
     <property name="minimumSize">
      <size>
       <width>211</width>
       <height>0</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Marke</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="5">
    <widget class="QComboBox" name="coMarke">
     <property name="minimumSize">
      <size>
       <width>211</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Hauptkat.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="5">
    <widget class="QComboBox" name="coHaupt">
     <property name="minimumSize">
      <size>
       <width>211</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Unterkat.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="5">
    <widget class="QComboBox" name="coUnter">
     <property name="minimumSize">
      <size>
       <width>211</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Gewicht</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="tbGroesse">
     <property name="minimumSize">
      <size>
       <width>61</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>       Einheit</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QComboBox" name="coEinheit">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>g</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ml</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>m</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Ablaufdatum</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="tbDatum">
     <property name="minimumSize">
      <size>
       <width>61</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>       Preis</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QLineEdit" name="tbPreis">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>€</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Anzahl </string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="tbAnzahl">
     <property name="minimumSize">
      <size>
       <width>61</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>       Soll</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4" colspan="2">
    <widget class="QLineEdit" name="tbSoll">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Lagerort</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="5">
    <widget class="QComboBox" name="coLager">
     <property name="minimumSize">
      <size>
       <width>211</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Brennwert</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="tbBrennwert">
     <property name="minimumSize">
      <size>
       <width>61</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>kJ   Kalorien</string>
     </property>
    </widget>
   </item>
   <item row="10" column="4">
    <widget class="QLineEdit" name="tbKalorien">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="10" column="5">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>kcal</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Protein</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="tbProtein">
     <property name="minimumSize">
      <size>
       <width>61</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>g   Fett</string>
     </property>
    </widget>
   </item>
   <item row="11" column="4">
    <widget class="QLineEdit" name="tbFett">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="11" column="5">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>g</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Kohlehydrate</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="tbKohle">
     <property name="minimumSize">
      <size>
       <width>61</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>g  Zucker</string>
     </property>
    </widget>
   </item>
   <item row="12" column="4">
    <widget class="QLineEdit" name="tbZucker">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="12" column="5">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>g</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>Liste</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="5">
    <widget class="QTableWidget" name="twListe">
     <property name="minimumSize">
      <size>
       <width>211</width>
       <height>0</height>
      </size>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>96</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>16</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>16</number>
     </attribute>
     <column>
      <property name="text">
       <string>Datum</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Anzahl</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Preis</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
